packages/kn-plugin-workflow/pkg/common/checks.go

```diff
@@ -135,7 +151,7 @@ func parseMavenVersion(version string) (int64, int64, error) {
 }
 
 func CheckIfDirExists(dirName string) (bool, error) {
```

Comment:
There is something wrong with the second `if`. It should not return `nil`, but rather more specific informations.